workflows/VGP-assembly-v2/Assembly-Hifi-HiC-phasing-VGP4/CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # Changelog
 
 
+## [0.1.9] 2024-04-19
+
+### Manual Updates
```

Comment:
For future updates, can you use the categories listed in https://keepachangelog.com/en/1.0.0/ ? I think in this case `Added` would have been the right headline.